### PR TITLE
CAYCPLUGIN-24 Implement BBT

### PR DESCRIPTION
**Pull request by Nolwenn-cadic-sonarsource**

### CODE REVIEW

File: .cirrus/bbt.sh
Comment (Hartesic) [inline .cirrus/bbt.sh:1]: Is this file necessary?

File: pom.xml [hunk 1]
```diff
@@ -176,7 +201,76 @@
...
+                  <executions>
+                      <execution>
+                          <id>run-black-box-tests</id>
+                          <goals>
+                              <goal>integration-test</goal>
+                              <goal>verify</goal>
+                          </goals>
+                          <configuration>
+                            <includes>
+                                  <include>**/*BBT.java</include>
```
Comment (Hartesic) [inline pom.xml:264]: What about:
```suggestion
                                  <include>**/bbt/*BBT.java</include>
```
to force BBTs to be in a specifically-named folder?

**Review by Hartesic — APPROVED** [review]
LGTM with FV

File: .cirrus/bbt.sh
Comment (Nolwenn-cadic-sonarsource) [reply .cirrus/bbt.sh:1]: Good catch, indeed this is not needed, I will remove it

File: pom.xml (hunk 1 above)
Comment (Nolwenn-cadic-sonarsource) [reply pom.xml:264]: If you believe this is better, we can change. I personally don't feel the need to change it, the first already works well. And if by mistake someone doesn't put their test outside of the bbt folder, we are sure to still run it.

Comment (Hartesic) [reply pom.xml:264]: I'm ok with this
